client/src/main/java/org/apache/celeborn/client/write/DataPusher.java

```diff
@@ -137,6 +138,7 @@ public void run() {
           }
         };
     pushThread.setDaemon(true);
+    pushThread.setUncaughtExceptionHandler(new ThreadExceptionHandler("DataPusher-" + taskId));
```

Comment:
Could `pushThread` use `ThreadUtils#newDaemonSingleThreadExecutor` instead of this way?